kernel/src/model/rule.rs

```diff
@@ -162,7 +162,7 @@ impl Rule {
         }
         let description = self
             .decode_description()
-            .unwrap_or(Some("invalid description".to_string()));
+            .unwrap_or_else(|_| Some("invalid description".to_string()));
```

Comment:
Refactor to avoid allocation in case the unwrap goes well. See this [clippy lint](https://rust-lang.github.io/rust-clippy/master/index.html#/or_fun_call).